server/cli/src/cli.rs

```diff
@@ -649,17 +649,27 @@ fn run_yarn_install(directory: &str) -> Result<(), Box<dyn std::error::Error>> {
         return Ok(());
     }
 
-    let status = Command::new("yarn")
-        .args(["install", "--cwd", &convert_dir, "--no-lockfile"])
-        .stdout(Stdio::inherit())
-        .stderr(Stdio::inherit())
-        .status()?;
-
-    println!("status {:?}", status);
-
-    if !status.success() {
-        eprintln!("Error: `yarn install` failed");
-        return Err("Failed to run yarn install".into());
+    let node_modules_path = Path::new(&convert_dir).join("node_modules");
+
+    if !node_modules_path.exists() {
+        let status = Command::new("yarn")
+            .args([
+                "install",
+                "--cwd",
+                &convert_dir,
+                "--no-lockfile",
```

Comment:
Ohh wow on second run it's much faster, personally I would optimise this by running in parallel, but I don't think we should do this especially since I would pivot from extism to BoaJS